image/cli/mascli/templates/pipelinerun.yaml

```diff
@@ -19,6 +19,10 @@ spec:
       value: '$ARTIFACTORY_USERNAME'
     - name: artifactory_apikey
       value: '$ARTIFACTORY_APIKEY'
+    - name: dev_mode_airgap
```

Comment:
Do we need to pass dev_mode into the pipeline?  How is it used inside the pipeline?  DEV_MODE flag should really only affect the logic in the CLI in terms of the interactive prompt and removing certain guardrails.